### ISSUE TITLE
Ability to explicitly refresh tree-view

### ISSUE THREAD

**Issue by stereosteve** [atom/tree-view#133]
When using with [sshfs](http://fuse.sourceforge.net/sshfs.html) the tree and git status can get out of sync.  Is it possible to add a key command to explicitly refresh the tree?

**Comment by robclancy** [comment]
How is this still a thing? Why do editors miss the smallest fucking things? Your editor is far from bug free so just add a damn refresh so if it does stop working we can simply click and fix it!

Pathetic.

**Comment by edjroot** [comment]
+1

I'm on Ubuntu 14.04. I installed jspm, then `jspm init` and `jspm install`ed some packages, all while Atom was running. jspm_packages didn't appear in the tree and neither of the suggested methods worked. I had to remove the root folder from the project and then add it again. This seems like a pretty basic feature that should be working at this point.

**Comment by mdeboer** [comment]
The tree also gets out of sync when using GIT if you change the gitignore file, a refresh button would be nice to fix this.

**Comment by 360art** [comment]
I got out of sync when rename folder. Window:reload do the trick but it is time-consuming.

**Comment by slikts** [comment]
My files are on a Samba share, and the tree view periodically goes out of sync. It would be nice if there was a command to refresh the tree view, as Window: Reload is painful to use.

**Comment by lnxg33k** [comment]
The same thing in here when working with sshfs mount.

**Comment by paramr** [comment]
Same here.  Having a shortcut (Cmd+r?) for this when on a tree node will help a lot.

**Comment by AndreHorst** [comment]
Yes, deleting adding files in the GIT repo doesnt change the color code correctly!

**Comment by mdeboer** [comment]
I think with all the issues it should autorefresh really. Anyway, I found that collapsing and opening a folder fixes the issue so it appears you don't need to re-draw the whole tree.

**Comment by AndreHorst** [comment]
collapsing and re-collapsing is not working for me. However, toggling the treeview did the trick.

**Comment by benjamingwynn** [comment]
A button or even a command in the command palette would be nice. Just for peace of mind if nothing else.

**Comment by jonilaserson** [comment]
Yep, also having this issue. Quickly gets out of sync with sshfs and there is no ability to refresh other than file->reopen project.

**Comment by mdeboer** [comment]
Just a thought:

Atom is colouring my tree view files and directories with colours corresponding with their git status. This means that it is monitoring files and folders for changes or polling `git status` (I haven't checked). Is it me or would this mean there's functionality there to watch for changes and trigger an action e.g. refresh the tree?

**Comment by ryanpatiency** [comment]
Having same issue

**Comment by exse2** [comment]
I had success by installing tree-view-auto-fold, binding this key in keymap.cson:
  'f10': 'tree-view-auto-fold:toggle'
and then toggling it off and on again.

**Comment by abhijit113** [comment]
> collapsing and re-collapsing is not working for me. However, toggling the treeview did the trick.

yeap....it worked
